test/runtests.jl

```diff
@@ -730,7 +730,7 @@ HISTORY this is a history"""
             @test getkey(inhdr, "INTKEY", nothing) == "INTKEY"
             @test get(inhdr, "BADKEY", nothing) === nothing
             @test get(inhdr, "INTKEY", nothing) == inhdr["INTKEY"]
-            @test get(() -> nothing, inhdr, "BADKEY") == nothing
+            @test get(() -> nothing, inhdr, "BADKEY") === nothing
```

Comment:
or `isnothing`?

Reply:
This form seems clearer to me, as it is easier to focus on the expression. However, I have no strong feelings either way.